examples/jetstream/ack-ack/dotnet2/Main.cs

```diff
@@ -0,0 +1,114 @@
+// Install NuGet packages `NATS.Net` and `Microsoft.Extensions.Logging.Console`.
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Configuration;
+using NATS.Client.Core;
+using NATS.Client.JetStream;
+using NATS.Client.JetStream.Models;
+using NATS.Client.KeyValueStore;
+
+using var loggerFactory = LoggerFactory.Create(builder => builder.AddConsole());
+var logger = loggerFactory.CreateLogger("NATS-by-Example");
+
+// `NATS_URL` environment variable can be used to pass the locations of the NATS servers.
+var url = Environment.GetEnvironmentVariable("NATS_URL") ?? "127.0.0.1:4222";
+
+// Connect to NATS server. Since connection is disposable at the end of our scope we should flush
+// our buffers and close connection cleanly.
+var opts = new NatsOpts
+{
+    Url = url,
+    LoggerFactory = loggerFactory,
+    Name = "NATS-by-Example",
+};
+await using var nats = new NatsConnection(opts);
+var js = new NatsJSContext(nats);
+
+// ### Stream Setup
+var stream = "verifyAckStream";
+var subject = "verifyAckSubject";
+var consumerName1 = "consumer1";
+var consumerName2 = "consumer2";
+
+// Remove the stream first!, so we have a clean starting point.
+try
+{
+    await js.DeleteStreamAsync(stream);
+}
+catch (Exception)
+{
+    // An exception here means the stream did not exist.
+}
+
+// Create the stream
+var streamConfig = new StreamConfig(stream, new[] { subject })
```

Comment:
```suggestion
var streamConfig = new StreamConfig(stream, [subject])
```
you can now use collection expressions since C# 12 / .NET 8

Reply:
done